### ISSUE TITLE
Add support for config files from URL

### ISSUE THREAD

**Issue by skundrik** [elastic/curator#654]
Now there is a support for config file it would be great to be able to get them from central store.

**Comment by untergeek** [comment]
It's on the roadmap to put them into Elasticsearch as a document, since ES can ingest YAML.  If someone wanted to add other ways/means of ingesting those files, I'm happy to merge PRs (must have tests).

**Comment by skundrik** [comment]
But you still need to bootstrap curator into particular ES in the first place before you can retrieve action file. Or am I missing something?

**Comment by untergeek** [comment]
Yes, but this is why the [client configuration file](https://www.elastic.co/guide/en/elasticsearch/client/curator/4.0/configfile.html) `curator.yml` file and the [action configuration file](https://www.elastic.co/guide/en/elasticsearch/client/curator/4.0/actionfile.html) `my_actions.yml` file are separate.  With a local configuration for the client present, then the actions can be stored in other locations, such as the same Elasticsearch cluster you're Curating.

Are you suggesting that pulling even the client configuration should be possible over URL?  That becomes a much more interesting question, especially since there could be authentication credentials in that file: user/pass, AWS tokens, etc.  You'd still need local access to any SSL certificates, if you were authenticating in that way, too.

**Comment by skundrik** [comment]
Yes, I was suggesting that even client configuration file might come over URL. Securing the connection/information is of course another problem that should be considered by the user. However this is less of an issue in dev/test environments.

**Comment by untergeek** [comment]
If someone wants to add it, and provide tests, then I'm happy to merge that.  I won't likely be adding it myself, though.
